synapse/storage/databases/main/room.py

```diff
@@ -1448,7 +1472,7 @@ def _background_populate_rooms_creator_column_txn(txn: LoggingTransaction):
         return batch_size
 
 
-class RoomStore(RoomBackgroundUpdateStore, RoomWorkerStore, SearchStore):
+class RoomStore(RoomBackgroundUpdateStore, RoomWorkerStore):
```

Comment:
`SearchStore` isn't needed as a parent class and makes Python fail to come up with an mro.